dashboard/new-dashboard/src/components/rubymine/PerformanceDashboard.vue

```diff
@@ -316,6 +364,14 @@
           'redmine-project-test/completion/qualified',
           'RUBY-23764-Case1/ruby-23764-findusages-case1',
           'RUBY-23764-Case2/ruby-23764-findusages-case2',
+          'RUBY-32357/module',
```

Comment:
Intentionally skipping adding class here as it's for the project model which is already included in this test.